source/modules/MEI.xml

```diff
@@ -2013,6 +2013,18 @@
       </valList>
     </content>
   </macroSpec>
+  <macroSpec ident="data.MENSURATIONSIGNNUMBERS" module="MEI" type="dt">
+    <desc>Number values as part of mensuration signs.</desc>
+    <content>
+      <rng:data type="token">
+        <rng:param name="pattern">[\d|Z]?[ [\d|Z]]*</rng:param>
+      </rng:data>
+    </content>
+    <remarks>
+      <p>Allows for multiple tokens to deal with signs containing multiple numbers, like "O 2 2" in Hothby’s “Ora pro nobis” (Codex Faenza fol. 28r).</p>
+      <p>The glyph "Z" used in 17th century Spanish manuscripts is treated as number 3.</p>
```

Comment:
Thanks for this PR, @annplaksin. A small correction regarding the interpretation of Z here, it should be: 
> The glyph "Z" used in 17th-century Spanish manuscripts is treated as the ratio 3/2.

Reply:
Thanks @martha-thomae !